src/video-grid/useMediaStream.ts

```diff
@@ -84,6 +85,13 @@ export const useMediaStream = (
     }
   }, [audioOutputDevice]);
 
+  useEffect(() => {
+    if (!mediaRef.current) return;
+    if (localVolume === null || localVolume === undefined) return;
```

Comment:
```suggestion
    if (localVolume == null) return;
```

This is probably just a personal preference so feel free to ignore